src/Controls/src/Core/Platform/GestureManager/GestureManager.Windows.cs

```diff
@@ -511,6 +519,81 @@ void OnPointerReleased(object sender, PointerRoutedEventArgs e)
 			PanComplete(true);
 		}
 
+		void OnPgrPointerEntered(object sender, PointerRoutedEventArgs e)
+		{
+			var view = Element as View;
+			if (view == null)
+				return;
+
+			var pointerPoint = e.GetCurrentPoint(Control);
+			var children = (view as IGestureController)?.GetChildElements(new Point(pointerPoint.Position.X, pointerPoint.Position.Y));
+
+			if (children != null)
+				foreach (var recognizer in children.GetChildGesturesFor<PointerGestureRecognizer>())
+				{
+					recognizer.SendPointerEntered(view);
+				}
```

Comment:
```suggestion
			if (children != null)
			{
				foreach (var recognizer in children.GetChildGesturesFor<PointerGestureRecognizer>())
				{
					recognizer.SendPointerEntered(view);
				}
			}
```

Add braces to this if. Personally I prefer to do it on any if, even one-liners, just so it's very clear but I don't think we have any hard agreements on that.